commons_api/settings.py

```diff
@@ -89,7 +90,8 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
-    'PAGE_SIZE': 1000
+    'PAGE_SIZE': 1000,
```

Comment:
Can this be made an environment variable?

Reply:
Done :-)